app/core/configs.py

```diff
@@ -44,8 +44,8 @@ class DetectorConfig(BaseModel):
             "Requires `always_return_edge_prediction=True`."
         ),
     )
-    min_time_between_escalations: Optional[float] = Field(
-        default=None,
+    min_time_between_escalations: float = Field(
+        default=2.0,
```

Comment:
Should the default value be 0.0? It seems like right now the default configuration will lead to a warning being logged.

Reply:
Fair point about the warning. Elsewhere we default to two if this isnt provided... And two seems like a sane safety number.

Reply:
The warning also isn't a huge deal, it's just warning the user that they might have misconfigured their detector. We could just let it be or change it to info. This will get solved at some point anyways when the config setup is redone.